src/Tools/dotnet-monitor/ConfigurationTokenParser.cs

```diff
@@ -35,15 +39,21 @@ internal sealed class ConfigurationTokenParser
         public const string Separator = ".";
 
         private const string ProcessInfoReference = "Process";
+        private const string ActionReference = "Action";
+
         private const string RuntimeId = "RuntimeId";
         private const string ProcessId = "ProcessId";
         private const string ProcessName = "Name";
         private const string CommandLine = "CommandLine";
+        private const string Hostname = "Hostname";
+        private const string UnixTime = "UnixTime";
 
         public static readonly string RuntimeIdReference = CreateTokenReference(ProcessInfoReference, RuntimeId);
         public static readonly string ProcessIdReference = CreateTokenReference(ProcessInfoReference, ProcessId);
         public static readonly string ProcessNameReference = CreateTokenReference(ProcessInfoReference, ProcessName);
         public static readonly string CommandLineReference = CreateTokenReference(ProcessInfoReference, CommandLine);
+        public static readonly string HostnameReference = CreateTokenReference(ProcessInfoReference, Hostname);
```

Comment:
As it is, this variable (`Process.Hostname`) is semantically referring to the hostname of the target process. But the callers are collecting the hostname information of the host that .NET Monitor is running on. While these might be the same in many cases, I think this should be either:
- properly classified with a new prefix (e.g. `Monitor.Hostname`) if using the hostname on which .NET Monitor is running
- or, get the environment block from the target process and resolve the host name from that. See https://github.com/dotnet/dotnet-monitor/blob/main/src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs#L178 as an example of how to get the environment block of the target process.

Reply:
Based on both your feedback, and my own proclivity, I would prefer adding a token whose behavior is consistent across all platforms. Otherwise, I think these sorts of caveats could become confusing. That is why in this latest iteration, I have settled on `Monitor.HostName`.